commons-db/src/main/java/com/palantir/nexus/db/manager/DatabaseConstants.java

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
not for you to do right now, but I'm guessing at some point in the future we'll either remove or go all-in for i18n in atlas code